src/main/java/CarRacing/view/InputView.java

```diff
@@ -0,0 +1,59 @@
+package CarRacing.view;
+
+import java.util.Scanner;
+
+import exception.NegativeInputException;
+
+public class InputView {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public int getNumberOfCar(String numberStr) {
+        if (emptyText(numberStr)) {
+            System.out.println("자동차 대수는 몇 대 인가요?");
+            numberStr = SCANNER.nextLine();
+        }
+
+        if (!isNumeric(numberStr)) {
+            throw new IllegalArgumentException("차량 대수는 숫자로만 입력해주세요!");
+        }
+
+        int numberOfCar = Integer.parseInt(numberStr);
+        if (numberOfCar < 0 ) {
+            throw new NegativeInputException("차량 대수는 양의 숫자만 입력해주세요!");
+        }
+
+        return numberOfCar;
+    }
```

Comment:
입력을 받아야하는 메서드인데 파라미터를 받을 필요가 있을까요?

```suggestion
    public int getNumberOfCar() {
        System.out.println("자동차 대수는 몇 대 인가요?");
        String numberStr = SCANNER.nextLine();

        try {
            int number = Integer.parseInt(numberStr);
            validate(number);
            return number;
        } catch (IllegalArgumentException e) {
            System.out.println("1 이상의 숫자만 입력해 주세요!");
            return getNumberOfCar();
        }
    }
```
재귀를 사용하면 문제를 해결할 수 있어보여요!